apiv1/impl-ecl.lisp

```diff
@@ -8,8 +8,12 @@ Distributed under the MIT license (see LICENSE file)
 
 (in-package #:bordeaux-threads)
 
+(eval-when (:compile-toplevel :execute)
+  (unless (<= ext:+ecl-version-number+ 210201)
```

Comment:
This test should check that `+ecl-version-number+` is >= than the first known version to support timeouts. Which one is that ?

Reply:
The version number is 230909